src/components/workspace/source.js

```diff
@@ -0,0 +1,119 @@
+ace.define("ace/theme/source",["require","exports","module","ace/lib/dom"], function(acequire, exports, module) {
```

Comment:
Can we add a comment indicating that this theme originated from the cobalt theme that we have been using. As well as comment on what changes we made?

Reply:
Okay done.